composer.json

```diff
@@ -107,7 +107,9 @@
     "doctrine/annotations": "^1",
     "squizlabs/php_codesniffer": "3.*",
     "exussum12/coverage-checker": "^0.11.2",
-    "micheh/phpcs-gitlab": "^1.1"
+    "micheh/phpcs-gitlab": "^1.1",
+    "jmikola/geojson": "^1.0",
```

Comment:
Unless I am mistaken, we aren't doing any geo-stuff on the backend (we don't have any polygons, etc, or any calculations to make).

Can you comment why we need these two libraries?

Reply:
Edit: I see why you need one of them, not the other :-)

Reply:
I have removed the latter.